src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -38,7 +38,7 @@ export const AccessApprovalsTable: React.FunctionComponent<
     [accessorId, accessRequirementId, sort],
   )
 
-  const { data, hasNextPage, fetchNextPage, isFetching } =
+  const { data, hasNextPage, fetchNextPage, isLoading } =
```

Comment:
`isLoading` is true whenever we are fetching data, and we don't have any data currently loaded
`isFetching` is true whenever data is being fetched (including when stale data is currently loaded)

`isLoading` is more appropriate for our spinners in these cases